ee/codegen/src/__test__/__snapshots__/project.test.ts.snap

```diff
@@ -657,7 +657,251 @@ class WeatherFunction(BaseWorkflow[Inputs, BaseState]):
 "
 `;
 
-exports[`WorkflowProjectGenerator > function > should generate inline workflow tool > code/workflow.py 1`] = `
+exports[`WorkflowProjectGenerator > function > should generate inline workflow tool 'no packages' > code/workflow.py 1`] = `
```

Comment:
Ideally, we add a test of smaller scope. Given deadline Thursday, moving a little bit quicker